config/locales/api/en.yml

```diff
@@ -76,241 +76,241 @@ en:
       transgender: "Transgender"
   help:
     menu:
-      videos: Videos
-      start: 1. Getting started
-      register: 2. Register a patient
-      search: 3. Find the correct patient
-      record: 4. Record a patient’s visit
-      overdue: 5. Call overdue patients
-      faq: Common questions
+      videos_html: Videos
```

Comment:
It's a Rails thing, actually. It auto-marks the keys as html-safe, without having to do `t("the.thing").html_safe`.

I thought initially that we'd be able to get away with just a few keys (the ones with spans or anchor tags embedded), but I didn't realize that some of the translations of plain strings could have HTML entities in them as well. If it was just fancy quotes, we could paste the special chars directly into the string, but it's funky things like `&zwnj;`. For safety, and since all this content is controlled by us and is intended to be displayed in HTML, I marked all the keys `_html`.

It was a tricky one, but I went with this approach.

I'm not worried about clobbering the keys in Transifex. There are auto-populating and suggestion features that make it easy to quickly re-translate the new keys if the old ones get borked.